src/main/scala/org/broadinstitute/dsde/workbench/sam/Boot.scala

```diff
@@ -80,7 +80,7 @@ object Boot extends IOApp with LazyLogging {
 
         _ <- dependencies.cloudExtensionsInitializer.onBoot(dependencies.samApplication)
 
-        binding <- IO.fromFuture(IO(Http().bindAndHandle(dependencies.samRoutes.route, "0.0.0.0", 8080))).onError {
+        binding <- IO.fromFuture(IO(Http().newServerAt("0.0.0.0", 8080).bind(dependencies.samRoutes.route))).onError {
```

Comment:
is this a change required due to dependency update?

Reply:
Yes, I should have added a link:  https://doc.akka.io/japi/akka-http/current/akka/http/javadsl/Http.html#bindAndHandle-akka.stream.javadsl.Flow-akka.http.javadsl.ConnectHttp-akka.stream.Materializer-

`Deprecated. since 10.2.0: Use Http.get(system).newServerAt(interface, port).bindFlow(handler) instead.`

Reply:
ah cool -- thanks!